third_party/workspace.bzl

```diff
@@ -30,12 +30,9 @@ def tensorboard_workspace():
 
   native.http_archive(
       name = "protobuf",
-      urls = [
-          "http://mirror.bazel.build/github.com/google/protobuf/archive/2b7430d96aeff2bb624c8d52182ff5e4b9f7f18a.tar.gz",
-          "https://github.com/google/protobuf/archive/2b7430d96aeff2bb624c8d52182ff5e4b9f7f18a.tar.gz",
-      ],
-      sha256 = "e5d3d4e227a0f7afb8745df049bbd4d55474b158ca5aaa2a0e31099af24be1d0",
-      strip_prefix = "protobuf-2b7430d96aeff2bb624c8d52182ff5e4b9f7f18a",
+      urls = ["http://mirror.bazel.build/github.com/google/protobuf/archive/0b059a3d8a8f8aa40dde7bea55edca4ec5dfea66.tar.gz"],
```

Comment:
Just a question - these dep URLs don't seem to indicate the version number anywhere in them.  Is that WAI?  It seems like it might be easy to get the wrong version's URL inadvertently.

Reply:
This is copy pasta from TensorFlow's workspace.bzl, which is actually in the process of being updated right now to track a version rather than a SHA. See cl/170858532

Reply:
Gotcha, thanks for clarifying.